app/controllers/change_requests_controller.rb

```diff
@@ -13,16 +13,28 @@ def create
   end
 
   def create_change_request
+
+    if params[:change_request][:action] == "0"
+      handle_insert
+    elsif params[:change_request][:action] == "1"
```

Comment:
Looks like we're going with the numbers (as strings) for the API. Should @azhelle16 also adopt this convention instead of the string "remove" in https://github.com/ShelterTechSF/askdarcel-api/pull/514 ?

Reply:
Changed this to be a string in https://github.com/ShelterTechSF/askdarcel-api/pull/520/commits/77e473c7706081d78a9261b966de991254158854.